### PR TITLE
Bumping gevent to resolve CVE-73655

### PR DESCRIPTION
**Pull request by pricem14pc**
# What and why?

Build failure following merge to main

> -> Vulnerability found in gevent version 24.2.1
>    Vulnerability ID: 73655
>    Affected spec: <24.10.1
>    ADVISORY: Affected versions of gevent are vulnerable to a Race
>    Condition leading to Unauthorized Access — CWE-362. The attack can be...
>    PVE-2024-[73](https://github.com/ONSdigital/ras-party/actions/runs/11774400065/job/32792885285#step:5:74)655
>    For more information, please visit
>    https://data.safetycli.com/v/73655/[74](https://github.com/ONSdigital/ras-party/actions/runs/11774400065/job/32792885285#step:5:75)2

# Jira

https://jira.ons.gov.uk/browse/RAS-1345

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ flask = "*"
 flask-cors = "*"
 flask-httpauth = "*"
 gunicorn = "*"
-gevent = {version = "*", platform_python_implementation="=='CPython'"}
+gevent = "*"
```
Comment (pricem14pc) [inline Pipfile:15]: This change was done by the `pipenv upgrade gevent` so I am assuming it's no longer required in the Pipfile. Looking at Github comments it does appear that this was a previous workaround to a previous issue.

**Review by arroyoAle — APPROVED** [review]
Looks good and party works, updates to gevent and connected dependencies

**Review by matthew-robinson-ons — APPROVED** [review]
Changes look good and party tests and function all working

**Pull request merged**